charts/hello/templates/tests/test-connection.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
 {{ include "hello.labels" . | nindent 4 }}
   annotations:
     "helm.sh/hook": test-success
+    policy.otomi.io/ignore: pspallowedusers
```

Comment:
I hope these hardcoded ignores will be "ignored" if psp-allowed-users is not enabled.

Reply:
I don't understand what you mean here. If the policy is disabled it is exempt from evaluation.
If you mean we should have conditionals before appending ignore annotations to charts.. I don't believe that is such a great idea, as we are interested in adjusting templates so that they are conforming to the security settings in the policies. Example ( adding **securityContext** / **privileged** / **hostPath** / etc)

Reply:
OK. it makes sense to have annotations only for used resources.. so I am writing a snippet function for this, calling it maybe `addIgnorePolicyAnnotation(["policyID","policyID"])`.

Reply:
ok, Done
we can include policy annotations with following templating syntax: 
```
  {{ tpl $addIgnorePolicyAnnotation (list "psp-allowed-users" "banned-image-tags") }}
```
If the policies are enabled they will be included in the list, otherwise if the list ends up empty, no annotation field will be provided.